### PR TITLE
add vf switcher to utilities

### PR DESCRIPTION
**Pull request by abdallahyas**
Added the ansible utility to deploy the vf-switcher service.

P.S. This PR depends on a commit from #113, so merging either of them will cause a conflict in the other and the other will need to be rebased.

### CODE REVIEW

File: ansible/projects/tasks/get_worker_nodes.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- name: get docker containers
+  shell:
+    cmd: docker ps -f name=worker --format "{{ '{{' }}.Names{{ '}}' }}"
```
Comment (Mmduh-483) [inline ansible/projects/tasks/get_worker_nodes.yml]: Use ` kind get  nodes --name <cluster>`

Comment (Mmduh-483) [reply ansible/projects/tasks/get_worker_nodes.yml]: Ovn-kubernetes ansible is doing same task, https://github.com/Mellanox/kubernetes-ci/blob/master/ansible/projects/ovn-kubernetes/tasks/deploy.yaml#L3 , can you make both read the same file to not redundant it

File: ansible/roles/kind/kind_cluster_destroy/tasks/stop_vf_switcher.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+- name: Stop the vf-switcher service
+  ansible.builtin.systemd:
```
Comment (Mmduh-483) [inline ansible/roles/kind/kind_cluster_destroy/tasks/stop_vf_switcher.yml]: I think it is safe to run it without prefix `ansbile.builtin`

File: ansible/roles/utilities/vf_switcher/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    dest: "/usr/local/bin/vf-netns-switcher.sh"
+    mode: +x
+
+- name: Copy vf-switcher-service file to systemd
+  copy:
+    src: "{{ playbook_dir }}/../deploy/vf-switcher.service"
+    dest: "/etc/systemd/system/vf-switcher.service"
+
+- name: Start vf-switcher service
+  ansible.builtin.systemd:
```
Comment (Mmduh-483) [inline ansible/roles/utilities/vf_switcher/tasks/main.yml]: same about ansible.builtin

**Review by Mmduh-483 — APPROVED** [review]

**Review by e0ne — APPROVED** [review]

**Pull request merged**